package.json

```diff
@@ -36,7 +36,7 @@
   },
   "dependencies": {
     "@aws-sdk/client-lambda": "^3.436.0",
-    "axios": "1.6.1",
+    "axios": "1.6.3",
```

Comment:
is there a reason why you are pinning this? If it was just `^1.6.0` the next install would pick up the latest

Reply:
Fix provided in #156